docs/content/en/docs/implementing/evaluations.md

```diff
@@ -3,3 +3,85 @@ title: Evaluations
 description: Understand Keptn evaluations and how to use them
 weight: 150
 ---
+A `KeptnEvaluation` is a `KeptnMetric` that has a defined target value.
+A `KeptnEvaluation` resource is defined in a
+[KeptnEvaluationDefinition](../yaml-crd-ref/evaluationdefinition.md)
+yaml file.
+The example
+[app-pre-deploy-eval.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/version-3/app-pre-deploy-eval.yaml)
+file specifies the `app-pre-deploy-eval-2` evaluation as follows:
+
+```yaml
```

Comment:
instead of hardcoding yamls we could embed existing examples https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/config/samples/lifecycle_v1alpha1_keptnevaluationdefinition.yaml
 
 so this will become: 
 
 {{< embed path="/lifecycle-operator/config/samples/lifecycle_v1alpha1_keptnevaluationdefinition.yaml" >}}

Reply:
Done, only I used the file under the examples folder using the v1alpha3 library.